stable/insights-agent/values.yaml

```diff
@@ -238,6 +238,33 @@ pluto:
       cpu: 10m
       memory: 64Mi
 
+prometheus:
```

Comment:
Can we rename this to `resource-metrics`? I don't want to give the impression we're _just_ installing p8s.

That'll also help us separate out the values that get passed to the p8s chart versus the values we use in the CronJob. I know we combine them with goldilocks, but it makes me a little squeamish. I wish helm let us decide how values get passed to the subchart

Reply:
Yeah that sounds good. I had bounced around a little between Prometheus, Prometheus Collector, and metrics. Standardizing on resource-metrics sounds good to me.